### ISSUE TITLE
How to get the First Model for CIFAR100

### ISSUE THREAD

**Issue by PatrickZH** [fmcp/EndToEndIncrementalLearning#6]
Thank you for sharing the code. I wonder the detail of obtaining the first (ResNet) model for the first 10 classes in CIFAR100. I tried to reimplement your code with Pytorch. However, even at the first step, the performance is only 85% (averaged on many random splits, with your augmentation strategy), but yours is nearly 90%. It is said that the first model is trained using ResNet-Matconvnet. Do you used any pretrained model (by finetuning) when train the first model?

**Comment by PatrickZH** [comment]
By the way, in the Fig. 4, it is interesting that, at the first step, all models w./w.o. data augmentation, achieved nearly the same performance. Does it mean that data augmentation does not contribute to the training of the first model?

**Comment by fmcp** [comment]
Hi,

I use ResNet-Matconvnet for training the first model. Note that I use random noise and L2 norm in the gradients. Maybe that’s the difference.

Regarding the data augmentation, since I use the base model trained with ResNet-Matconvnet, it doesn’t use my data augmentation.

**Comment by PatrickZH** [comment]
Thank you for the reply! I will keep trying it.
